parallel-install/pkg/prerequisites/prerequisites.go

```diff
@@ -12,7 +17,17 @@ import (
 
 const logPrefix = "[prerequisites/prerequisites.go]"
 
-func InstallPrerequisites(ctx context.Context, prerequisites []components.Component, kubeClient kubernetes.Interface) <-chan error {
+//InstallPrerequisites tries to install all provided prerequisites.
+//The function quits on first encountered error, because all prerequisites must be installed in order to start Kyma installation.
```

Comment:
```suggestion
//The function quits on the first encountered error because all prerequisites must be installed in order to start Kyma installation.
```